languages/elixir/exercises/concept/access/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+## Learning objectives
+
+- Know of the access behavior
+- Know that the access behavior can be used on maps and keyword lists
+- Know the syntax for the access behavior
+- limitations of the access behavior
+  - doesn't work for structs
+  - doesn't raise errors in some circumstances
+
+## Out of scope
+
+- structs
+- static access operator
+
+## Concepts
+
+- `map-access-behavior`
```

Comment:
I think we could just name the concept `access-behavior`. It's not any different for keyword lists, so it doesn't make sense to have a separate exercise for `keyword-list-access-behavior`. What do you think?

Reply:
I think I did in the config.json, I think this was just missed.